### ISSUE TITLE
`docker volume -h` emits help message twice

### ISSUE THREAD

**Issue by micahhausler** [docker/docker#20093]
Description of problem:
    `docker volume -h` emits help message twice.

`docker version`:

```
$ docker version
Client:
 Version:      1.10.0
 API version:  1.22
 Go version:   go1.5.3
 Git commit:   590d5108
 Built:        Thu Feb  4 18:18:11 2016
 OS/Arch:      darwin/amd64

Server:
 Version:      1.10.0
 API version:  1.22
 Go version:   go1.5.3
 Git commit:   590d5108
 Built:        Thu Feb  4 19:55:25 2016
 OS/Arch:      linux/amd64
```

`docker info`:

```
$ docker info
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 113
Server Version: 1.10.0
Storage Driver: aufs
 Root Dir: /mnt/sda1/var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 115
 Dirperm1 Supported: true
Execution Driver: native-0.2
Logging Driver: json-file
Plugins: 
 Volume: local
 Network: null host bridge
Kernel Version: 4.1.17-boot2docker
Operating System: Boot2Docker 1.10.0 (TCL 6.4.1); master : b09ed60 - Thu Feb  4 20:16:08 UTC 2016
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 996.1 MiB
Name: default
ID: RLM2:AUEA:NZ33:CNQQ:3TN3:XRXE:KSAM:SS4S:BFSS:A6H2:V24A:MLSG
Debug mode (server): true
 File Descriptors: 10
 Goroutines: 23
 System Time: 2016-02-07T23:40:36.778891002Z
 EventsListeners: 0
 Init SHA1: 
 Init Path: /usr/local/bin/docker
 Docker Root Dir: /mnt/sda1/var/lib/docker
Username: micahhausler
Registry: https://index.docker.io/v1/
Labels:
 provider=virtualbox
```

`uname -a`:

```
Darwin ip-10-101-71-37.ec2.internal 14.5.0 Darwin Kernel Version 14.5.0: Tue Sep  1 21:23:09 PDT 2015; root:xnu-2782.50.1~1/RELEASE_X86_64 x86_64
```

Environment details (AWS, VirtualBox, physical, etc.):
VirtualBox on OS X

How reproducible:
100%

Steps to Reproduce:
1. `docker volume -h`
2. Enjoy

Actual Results:

```
$ docker volume -h 

Usage:  docker volume [OPTIONS] [COMMAND]

Manage Docker volumes

Commands:
  create                   Create a volume
  inspect                  Return low-level information on a volume
  ls                       List volumes
  rm                       Remove a volume

Run 'docker volume COMMAND --help' for more information on a command

  --help             Print usage

Usage:  docker volume [OPTIONS] [COMMAND]

Manage Docker volumes

Commands:
  create                   Create a volume
  inspect                  Return low-level information on a volume
  ls                       List volumes
  rm                       Remove a volume

Run 'docker volume COMMAND --help' for more information on a command

  --help             Print usage
flag: help requested
```

Expected Results:
It should be the same as `docker volume --help`

Additional info:

**Comment by runcom** [comment]
`docker network -h` is having the same issue actually

**Comment by runcom** [comment]
got a patch for this

**Comment by DrStrangepork** [comment]
This is still in 1.10.3

**Comment by thaJeztah** [comment]
Yes, still in 1.11 as well; looks like we never removed the `-h` as was discussed in https://github.com/docker/docker/issues/20094

**Comment by thaJeztah** [comment]
This was resolved in docker 1.12;

```
docker volume -h
Flag shorthand -h has been deprecated, please use --help

Usage:	docker volume COMMAND

Manage Docker volumes

Options:
      --help   Print usage

Commands:
  create      Create a volume
  inspect     Display detailed information on one or more volumes
  ls          List volumes
  rm          Remove one or more volumes

Run 'docker volume COMMAND --help' for more information on a command.
```
